weaviate/collections/aggregations/aggregate.py

```diff
@@ -77,6 +88,82 @@ def _to_aggregate_result(
                 f"There was an error accessing the {e} key when parsing the GraphQL response: {response}"
             )
 
+    def _to_result(
+        self, response: aggregate_pb2.AggregateReply, is_group_by: bool
+    ) -> Union[AggregateReturn, AggregateGroupByReturn]:
+        if len(response.result.groups) == 0:
+            raise WeaviateQueryError("No results found in the aggregation query!", "gRPC")
+        if is_group_by:
+            AggregateGroupByReturn(
```

Comment:
you don't assign or return this? Seems a duplicate of line 119+ and I think it should be returned here

Reply:
I have no idea why I've left that in as duplicated. Perhaps I meant to have this branch return but then, because it didn't, needed the final return statement. Will fix!